src/components/UI/TransferLogContainer/TransferLogContainer.js

```diff
@@ -21,23 +21,30 @@ export const TransferLogContainer = ({transferIndex, children, onShowTransfers})
   const renderChildren = () => {
     if (!children) {
       return <div className={styles.empty}>{emptyMsgTxt}</div>;
-    } else if (showChildren || transferIndex > -1) {
-      return children;
     }
 
     return (
       <div className={styles.viewMore}>
-        {Array.isArray(children) ? children.length : 1} {overviewTxt}{' '}
-        <span onClick={toggleShowChildren}>{viewMoreTxt}</span>
+        {Array.isArray(children) && children.length > 1
+          ? `${children.length} ${overviewTxt}`
+          : `1 ${singleOverviewTxt}`}{' '}
```

Comment:
Why do we need to define another  text?

Reply:
as we want to hold the strings outside the component (for i18n in future I guess)

Reply:
OK.
I still don't think it's a good solution but anyway we need to move to real i18n and then it will be managed by it.